modules/SFM/SFM.cpp

```diff
@@ -400,10 +400,18 @@ bool SFM::updateModule()
 
     if (outDisp.getOutputCount()>0)
     {
+
+        std::cout << "WITHIN OUTDISP.GETOUC" << std::endl;
+
         outputDm = stereo->getDisparity();
 
         if (!outputDm.empty())
         {
+
+            std::cout << "OUTPUT SIZE AND CHANNELS" << std::endl;
+            std::cout << outputDm.size() << std::endl;
+            std::cout << outputDm.channels() << std::endl;
+
```

Comment:
Better off to disable these printouts that may consume resources at run time.

Reply:
Done straight in @damianomal 's branch.

Reply:
Thank you @pattacini, those where some debug printouts I introduced in the former SFM code for debugging purposes, it's better for them to be gone!